docs/debug.md

```diff
@@ -0,0 +1,192 @@
+# Debugging SOCI snapshotter
+
+This document outlines where to find/access logs and metrics for the snapshotter. It attempts to provide some common error paths that a user might run into while using the snapshotter and provides some guidance on how to either root-cause the issue or resolve it.
+
+- [Debugging SOCI snapshotter](#debugging-soci-snapshotter)
+- [Finding Logs / Metrics](#finding-logs--metrics)
+  - [Logs](#logs)
+  - [Metrics](#metrics)
+    - [Accessing Metrics](#accessing-metrics)
+    - [Metrics Emitted](#metrics-emitted)
+- [Common Scenarios](#common-scenarios)
+  - [Rpull](#rpull)
+    - [No lazy-loading](#no-lazy-loading)
+    - [Rpull taking an abnormal amount of time](#rpull-taking-an-abnormal-amount-of-time)
+    - [Background Fetching](#background-fetching)
+  - [Running container](#running-container)
+    - [FUSE Read Failures](#fuse-read-failures)
+- [Debugging Tools](#debugging-tools)
+  - [CLI](#cli)
+  - [CPU Profiling](#cpu-profiling)
```

Comment:
Headers need to have consistent case format. Either capitalize only the first word or every word, but need to be consistent.